core/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala

```diff
@@ -781,7 +781,7 @@ class SQLPlanParserSuite extends BaseTestSuite {
     }
   }
 
-  test("Expression not supported in Generate") {
+  test("Expression supported in Generate") {
```

Comment:
nit: Since we are removing the comment - `json_tuple is not supported`, can we rename the test to specify which expression are we testing here?